React/Views/RCTModalHostView.m

```diff
@@ -89,6 +101,13 @@ - (void)insertReactSubview:(UIView *)subview atIndex:(NSInteger)atIndex
   RCTAssert(_reactSubview == nil, @"Modal view can only have one subview");
   [super insertReactSubview:subview atIndex:atIndex];
   [_touchHandler attachToView:subview];
+#if TARGET_OS_TV
+  if (_onRequestClose) {
```

Comment:
`onRequestClose` can be set later, right? So, probably we don't need this check here.